content/sensu-go/5.5/reference/agent.md

```diff
@@ -112,6 +112,35 @@ HTTP/1.1 202 Accepted
 
 _PRO TIP: You can use the agent API `/events` endpoint to create proxy entities by including a `proxy_entity_name` attribute within the `check` scope._
 
+#### Detecting silent failures
+
+You can use the Sensu agent API in combination with the check time-to-live attribute (TTL) to detect silent failures, creating what's commonly referred to as a "dead man's switch" (source: [Wikipedia][20]).
+By using check TTLs, Sensu is able to set an expectation that a Sensu agent continues to publish events for a check at a regular interval.
+If a Sensu agent fails to publish an event and the check TTL expires, Sensu creates a event with a status of `1` (warning) to indicate the silent failure.
+For more information on check TTLs, see the [the check reference][44].
+
+A great use case for the Sensu agent API is to create a dead man's switch to ensure that backup scripts continue to run successfully at regular intervals.
```

Comment:
```suggestion
A great use case for the Sensu agent API is to enable tasks which run outside of Sensu's check scheduling to emit check results. Using the Check TTL attribute, these results create a dead man's switch, ensuring that if the task fails for any reason, the lack of an "all clear" check result from the task will notify operators of a silent failures which might otherwise be missed.
```